### PR TITLE
feat: add notyet (under construction) page

### PR DESCRIPTION
**Pull request by Aashishkushwaha**
fix issue #14 - create under construction page.

Please review & merge the PR if everything looks good.

### CODE REVIEW

File: src/pages/notyet.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+};
+
+// markup
+const NotFoundPage = () => {
+  return (
+    <main
+      style={pageStyles}
+      className="bg-gradient-to-b from-green-400 via-blue-500 to-indigo-600"
+    >
+      <title>Under construction page</title>
```
Comment (capnkenny) [inline src/pages/notyet.js:44]:
```suggestion
      <title>NovelRT | Under Construction</title>
```
Suggestion here - I wouldn't explicitly state its the Under Construction page here but instead just mention it's "under construction"

File: src/pages/notyet.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        </p>
+        <Link to="/" style={linkStyles}>
+          Go home
+        </Link>
+      </div>
+    </main>
+  );
+};
+
+export default NotFoundPage;
```
Comment (capnkenny) [inline src/pages/notyet.js:62]: This will need to match what the page is renamed to like in the above comment.

File: src/styles/global.css [hunk 3]
```diff
@@ -3,10 +3,16 @@
...
-}
\ No newline at end of file
+  margin-right: 0px !important;
+}
+
+@media screen and (max-width: 600px) {
+  html {
+    font-size: 75%;
+  }
+}
```
Comment (capnkenny) [inline src/styles/global.css:18]: I'm curious - why was this added?

File: src/pages/notyet.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    >
+      <title>Under construction page</title>
+      <div style={contentStyle}>
+        <h1 style={brandStyles}>Under Construction</h1>
+        <p style={messageStyles}>
+          Sorry{" "}
+          <span role="img" aria-label="Pensive emoji">
+            😔
+          </span>{" "}
+          This page is still under construction
```
Comment (capnkenny) [inline src/pages/notyet.js:52]:
```suggestion
          This page is still under construction.
          We'll bring you new content as soon as it's ready!
```

Suggestion here - we'd like to have viewers be waiting for what is coming, so we should probably add the second line here (should probably be centered as well). Also, there was missing punctuation.

File: src/pages/notyet.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        </p>
+        <Link to="/" style={linkStyles}>
+          Go home
```
Comment (capnkenny) [inline src/pages/notyet.js:55]:
```suggestion
          Go Home
```
Suggestion - capitalize the H?

File: src/pages/notyet.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+const linkStyles = {
+  marginTop: "2rem",
+  display: "inline-block",
+  padding: ".8rem 4rem",
+  textAlign: "center",
+  background: "#374151",
+  borderRadius: "5rem",
+  fontSize: "1.4rem",
+  filter: "drop-shadow(0 0 10px #374151)",
+};
```
Comment (capnkenny) [inline src/pages/notyet.js:35]: Once the layout is added, this will probably either be changed or not needed and removed.

**Review by capnkenny — CHANGES_REQUESTED** [review]
Overall not a bad start - we'd like to see this page fall in line with the pages that are finished however. Let me know if you have any questions on any of the comments provided 👍

**Comment by capnkenny** [comment]
Hi @Aashishkushwaha , just curious to know if you've gotten a chance to review the requested changes yet?

Also, I did want to point out that in order for us to accept any contributions, our CLA _must_ be signed. Without sign off, we cannot accept the PR.

Thanks!

Cc @RubyNova

**Comment by Aashishkushwaha** [comment]
Hi @capnkenny, I'm working on the requested changes :)

File: src/pages/notyet.js [hunk 7]
```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import { Link } from "gatsby";
+
```
[the remark this answers is not in the document]
Comment (Aashishkushwaha) [reply src/pages/notyet.js:3]: made required change

File: src/pages/notyet.js [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+  padding: ".8rem 4rem",
+  textAlign: "center",
+  background: "#374151",
+  borderRadius: "5rem",
+  fontSize: "1.4rem",
+  filter: "drop-shadow(0 0 10px #374151)",
+};
+
+// markup
+const NotFoundPage = () => {
```
[the remark this answers is not in the document]
Comment (Aashishkushwaha) [reply src/pages/notyet.js:38]: made required change

File: src/pages/notyet.js (hunk 1 above)
Comment (Aashishkushwaha) [reply src/pages/notyet.js:44]: made required change

File: src/styles/global.css (hunk 3 above)
Comment (Aashishkushwaha) [reply src/styles/global.css:18]: this is to add responsiveness to website.

File: src/pages/notyet.js (hunk 4 above)
Comment (Aashishkushwaha) [reply src/pages/notyet.js:52]: made requested changes

File: src/pages/notyet.js (hunk 5 above)
Comment (Aashishkushwaha) [reply src/pages/notyet.js:55]: made requested change.

File: src/pages/notyet.js (hunk 2 above)
Comment (Aashishkushwaha) [reply src/pages/notyet.js:62]: made requested change

File: src/pages/notyet.js (hunk 6 above)
Comment (Aashishkushwaha) [reply src/pages/notyet.js:35]: made requested changes.

**Comment by Aashishkushwaha** [comment]
Hi @capnkenny , I've made all the requested changes. Please review changes & merge the branch.

**Comment by RubyNova** [comment]
Hi @Aashishkushwaha

Have you signed the CLA yet? I'm glad to see you've addressed his feedback, but like Kenny said, we require our CLA to be signed before we can actually merge anything.

**Comment by Aashishkushwaha** [comment]
Hi @RubyNova , Can you please help me in singing CLA, because I don't know what to do & how to sign CLA. It's my first time.

**Comment by RubyNova** [comment]
Sure no no worries. You just need to click on either the badge or the link to our CLA which you can find above in this comment https://github.com/novelrt/novelrt.github.io/pull/16#issuecomment-932366539

One of those two options will take you to our CLA, where you just need to fill out the relevant information and agree to it. Hope that helps. :)

**Comment by Aashishkushwaha** [comment]
Hi @RubyNova thanks for your help, I've signed the CLA

**Review by Aashishkushwaha — COMMENTED** [review]
I've completed all the required changes.

**Comment by capnkenny** [comment]
Thanks @Aashishkushwaha - I'll review as soon as possible!

**Review by capnkenny — APPROVED** [review]
Looks good to me - thank you for your contribution @Aashishkushwaha !
